safe/definitions/minimum_needs.py

```diff
@@ -70,9 +68,9 @@ def _initializes_minimum_needs_fields():
             field_name = '{namespace}__{field_name}'.format(**format_args)
             # It's easier to debug if we have the real value from the
             # postprocessor. That's why I put a Double. ET.
-            field_type = QVariant.Double
+            field_type = qvariant_numbers
```

Comment:
So I guess you can remove my two comment lines just above, but it will not be true anymore.